Sources/PenguinStructures/FactoryInitializable.swift

```diff
@@ -0,0 +1,65 @@
+//******************************************************************************
+// Copyright 2020 Penguin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+/// Classes having initializers that actually create derived classes.
```

Comment:
Non-blocking request: it'd be good to either: (1) flesh this out into a more complete example in the documentation, or (2) point readers to the corresponding tests to understand how to fit everything together.

Reply:
Please open an issue, or even better yet, "suggest" the exact text that should be added.  I thought this was pretty darned complete